### PR TITLE
Appointments

### PR DESCRIPTION
**Pull request by Andy-Hoch**
Appoointment Controller and First View here... Pleas review

### CODE REVIEW

File: app/controllers/appointments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      end
+    end
+  end
+
+  def new
+    @appointment = Appointment.new
+  end
+
+  def create
+    @appointment = Appointment.new(appointment_params)
```
[the remark this answers is not in the document]
Comment (Andy-Hoch) [reply app/controllers/appointments_controller.rb:25]: should be done

**Review by froidautaj — CHANGES_REQUESTED** [review]

File: app/views/appointments/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  <h1>Your Appointments</h1>
+  <ul>
+    <% @appointments.each do |app| %>
+      <li>
+        User: <%= app.user.first_name %> <%= app.user.last_name %> - <%= app.comment %> - <%= app.status %>
+        Provider: <%= app.treatment.user.first_name %> <%= app.treatment.user.last_name %> - <%= app.treatment.name %> - <%= app.status %>
```
[the remark this answers is not in the document]
Comment (Andy-Hoch) [reply app/views/appointments/index.html.erb:8]: :)

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root to: "treatments#index"
-  resources :treatments, only: %i[show]
-  resources :reviews, only: [:new, :create] # new is temporary
```
Comment (Hywel-L-Jones) [inline config/routes.rb:4]: As discussed, reviews should be nested within treatments, right?

**Review by Andy-Hoch — COMMENTED** [review]
Problems shoud be resolved

**Review by froidautaj — APPROVED** [review]

**Pull request merged**